src/test/static/class/class-properties.js

```diff
@@ -0,0 +1,25 @@
+class Class {
+  /** @public a static property with a getter */
```

Comment:
does this require the `@public` annotation to be picked up?

Reply:
it doesn't, i think it will just be used for setting `privacy` on the scanned property :)

Reply:
hmm it seems they are only picked up if theres at least one jsdoc comment (`@something`). so `/** foo */` doesn't yet work (@rictic commented about this above somewhere too, saying maybe we should still pick them up).